pgctl/cli.py

```diff
@@ -67,17 +58,48 @@ def svc(*args):
 
 def svstat(*args):
     # svstat *always* exits with code zero...
-    p = Popen(('svstat',) + tuple(args), stdout=PIPE)
-    status, _ = p.communicate()
+    cmd = ('svstat',) + tuple(args)
+    p = Popen(cmd, stdout=PIPE)
+    status_group, _ = p.communicate()
+
+    state_group = []
+    #status is listed per line for each argument
+    for status in status_group.splitlines():
+        state_group.append(get_state(status))
+
+    return state_group
```

Comment:
might be nicer as list-comp

status, _ = p.communicate()

return [
get_state(statusline)
for statusline in status.splitlines()
]